WickedEngine/wiHelper.cpp

```diff
@@ -1440,8 +1440,8 @@ namespace wi::helper
 #else
 		std::mbstate_t state = std::mbstate_t {};
 		const std::size_t len = std::mbsrtowcs(to, &from, dest_size_in_characters, &state);
-		if (len > 0) return len + 1; // string length + null terminating character
-		return len;
+		if (len == static_cast<std::size_t>(-1)) return len + 1; // string length + null terminating character
```

Comment:
shouldn't that be `>=`?

Reply:
not >= won't work because it's going to do a comparison between size_t and size_t which are unsigned. Why is it returning a negative number inside an unsigned? I have no idea, but it is certainly a debatable choice.